couchpotato/core/media/movie/providers/info/couchpotatoapi.py

```diff
@@ -102,12 +106,141 @@ def getInfo(self, identifier = None, adding = False, **kwargs):
 
         return {}
 
+
     def getReleaseDate(self, identifier = None):
         if identifier is None: return {}
 
         dates = self.getJsonData(self.urls['eta'] % identifier, headers = self.getRequestHeaders())
-        log.debug('Found ETA for %s: %s', (identifier, dates))
 
+        #This grabs release date info from omdbapi/rottentomatoes
+        temp2 = self.getJsonData("http://www.omdbapi.com/?i=%s&tomatoes=true&plot=short&r=json" % identifier)
+        title = temp2['Title']
+        year = int(temp2['Year'])
+
+        ddate=0 #throw away what couchpotatoai is returning since it is garbage at this time
+        tdate=0
+        dvd_date= temp2['DVD']
+        theater_date=temp2['Released']
+        if theater_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            tdate=int(time.mktime(time.strptime(theater_date,p)))
+
+        if dvd_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            ddate=int(time.mktime(time.strptime(dvd_date,p)))
+           
+        if (ddate !=0):    
+            if ddate < tdate:
+                ddate = 0 #if the dvd release date occurs before the theater release date, assume the data is wrong
+                tdate = 0
+
+        if (ddate or tdate):
```

Comment:
ok, i implemented some changes and briefly tested... could you try it out and see if it resolves your issue.. 

The way I implemented it now reflects the following:
        # if OMDBAPI returns both ddate and tdate use that and stop
        # if OMDBAPI returns partial or no information then check THEMOVIEDB
        # if THEMOVBIEDB returns both ddate and tdate use that and stop
        # if THEMOVIEDB returns partial or no information but OMDBAPI returns partial info use OMDBAPI's partial info and stop
        # if OMDBAPI returns no information and THEMOVIEDB returns partial information use that and stop
        # if both OMDBAPI and THEMOVIEDB return no information, no information is propagated and we stop.
        # could add some code in here to merge the two.. i.e. if one has dvd info and the other has theater info...
        # but ignore that case for now.

OMDBAPI takes precendence.. but full information takes precedence over partial information and there is no constructive merging of info from both sources... (i.e. to make full information out of two pieces of partial information)...

your suggestion regarding immediately accepting partial information if the theatrical release date is in the future is a good one.. (and i was about to implement it; however, i think its better to just check anyway, since if the theatrical release date is incorrect, and THEMOVIEDB has full information, it would be better) -- plus im too lazy to go in and add that check right now.. might do it in future if i have the time.... 

Please let me know if you have any further suggestions.